test/fixtures/users.yml

```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
Don't do all test practice in the tutorial, it is no need
you can delete test file